views/components/Sidebar.jsx

```diff
@@ -29,36 +30,14 @@ const Sidebar = ({
       alert(error.message)
     }
   }
-
   return (
     <div className={classes.sideBar}>
-      <Hidden mdUp>
-        <Drawer
-          variant="temporary"
-          anchor={theme.direction === 'rtl' ? 'right' : 'left'}
-          open={mobileOpen}
-          onClose={handleDrawerToggle}
-          classes={{
-            paper: classes.drawerPaper,
-          }}
-          ModalProps={{
-            keepMounted: true, // Better open performance on mobile.
-          }}
-        >
-          <DrawerComponent
-            classes={classes}
-            onLogout={handleLogout}
-            totalStudies={totalStudies}
-            totalSubjects={totalSubjects}
-            totalDays={totalDays}
-            user={user}
-          />
-        </Drawer>
-      </Hidden>
-      <Hidden smDown implementation="css">
+      <Hidden>
         <Drawer
-          variant="permanent"
-          open
+          variant={onPersist}
```

Comment:
`onPersist` is named as a click handler but it is passed as the variant. Should we name this prop `drawerVariant`? Does it ever change? I'm wondering if we even need the prop at all.

Reply:
It does change, when we are in the matrix page we do not want to display the sidebar. [2c26c5e](https://github.com/AMP-SCZ/dpdash/pull/352/commits/2c26c5ef82ff63420783fb443c4cd7a14df00535)